batcher/aligned/src/main.rs

```diff
@@ -56,8 +59,21 @@ pub enum AlignedCommands {
     DepositToBatcher(DepositToBatcherArgs),
     #[clap(about = "Get user balance from the batcher", name = "get-user-balance")]
     GetUserBalance(GetUserBalanceArgs),
-    #[clap(about = "Get user nonce from the batcher", name = "get-user-nonce")]
+    #[clap(
+        about = "Gets user current nonce from the batcher. This is the nonce you should send in your next proof.",
+        name = "get-user-nonce"
+    )]
     GetUserNonce(GetUserNonceArgs),
+    #[clap(
+        about = "Gets the user nonce directly from the BatcherPaymentService contract. Useful for validating the on-chain state and check if your transactions are pending in the batcher.",
+        name = "get-user-nonce-from-ethereum"
+    )]
+    GetUserNonceFromEthereum(GetUserNonceFromEthereumArgs),
+    #[clap(
+        about = "Gets the amount of proofs a user has queued in the Batcher.",
+        name = "get-amount-of-proofs-in-batcher-queue"
```

Comment:
Should be `get-amount-of-user-proofs-in-batcher-queue`, for consistency.

Reply:
My 2 cents `get-users-number-of-queued-proofs` :
1.) `number` > `amount` as we return a numeric value.
2.) `batcher-queue` -> `queued`, where we get the number (batcher) doesn't matter as much what we are fetching the `queued proofs` also past tense conveys the proofs previously submitted by the user.

Reply:
Yes, it's a nit, but I think it improves the CLI.
I like the `get-user-number-of-queued-proofs` version, in singular.

Reply:
¡dirigido!